lib/lita/handlers/locker_labels.rb

```diff
@@ -12,9 +12,10 @@ class LockerLabels < Handler
       include ::Locker::Resource
 
       route(
-        /^locker\slabel\slist#{COMMENT_REGEX}$/,
+        /^locker\slabel\slist#{COMMENT_REGEX}/,
```

Comment:
Should we even have `COMMENT_REGEX` here? I'm not sure in what case you would add a comment to the line, and it makes me less certain of how adding a comment would interact with the `--page` keyword 
argument.

Speaking of which, could you describe the `--page` functionality in the `help` text?

Reply:
I'm 👍on removing the comment regex from the list command, it was only recently merged to master and hasn't been in a release yet: https://github.com/esigler/lita-locker/pull/52)